FlashTxx.h

```diff
@@ -62,7 +62,8 @@
   #define FLASH_SIZE		(0x800000)		// 8MB
   #define FLASH_SECTOR_SIZE	(0x1000)		// 4KB sector size
   #define FLASH_WRITE_SIZE	(4)			// 4-byte/32-bit writes    
-  #define FLASH_RESERVE		(4*FLASH_SECTOR_SIZE)	// reserve top of flash 
+  //#define FLASH_RESERVE		(4*FLASH_SECTOR_SIZE)	// reserve top of flash
+  #define FLASH_RESERVE		(64*FLASH_SECTOR_SIZE)	// reserve top of flash

```

Comment:
This was the fix: changing reserve to 64 sectors.

Reply:
I would just go ahead and delete out the commented out FLASH_RESERVE definition. We can always use version control to go back if we need to.

Reply:
Pushed an update.